src/modules/poolModule.ts

```diff
@@ -983,4 +984,37 @@ export class PoolModule implements BaseModule {
       console.error(e);
     }
   }
+
+  public async fetchRoute(sourceToken: string, targetToken: string, amount: bigint) {
```

Comment:
Rename to fetchBestRoute. What do you think if we move the function to RouteModule?

Reply:
{sourceToken: string, targetToken, amount, pools?: Extended[], tokens: []}